scripts/run_nodes.sh

```diff
@@ -54,7 +54,7 @@ validator_ids_string="${validator_ids_string//${IFS:0:1}/,}"
 echo "Bootstrapping chain for nodes 0..$((N_VALIDATORS - 1))"
 ./target/release/aleph-node bootstrap-chain --raw --base-path "$BASE_PATH" --account-ids "$validator_ids_string" --chain-type local > "$BASE_PATH/chainspec.json"
 
-for i in $(seq "$N_VALIDATORS" "$(( N_VALIDATORS + N_NON_VALIDATORS - 1 ))"); do
+for i in $(seq 0 "$(( N_VALIDATORS + N_NON_VALIDATORS - 1 ))"); do
```

Comment:
This actually works as-is, since we enter that loop when `-n` is specified, e.g.
``` 
/scripts/run_nodes.sh -v 2 -n 2 -b false
Bootstrapping chain for nodes 0..1
Bootstrapping node 2
{
  "account_id": "5Dfis6XL8J2P6JHUnUtArnFWndn62SydeP8ee8sG2ky9nfm9",
  "aura_key": "5HbJUsEb8ZEDUMgQVEybiWD3Thir1LpgZncrcr5nyrX6XhLs",
  "aleph_key": "5CgdS9GqQgEmpJjAsH7YQJyk6zh8pNNLvmuLbvT9pjuxeZ1m",
  "peer_id": "12D3KooWNwLnrb9uonDJfMVo5Etf255JyHyto3FLSeGYcLrHEv99"
}
Bootstrapping node 3
{
  "account_id": "5F4H97f7nQovyrbiq4ZetaaviNwThSVcFobcA5aGab6167dK",
  "aura_key": "5DiBRpaAWUiXjSdKpvHaFHZ3Fx2jT5Erv14KM4wkRgdwkA41",
  "aleph_key": "5CR3ZPU3xtXkhVXu1yoSPW48iNdTR5qRxBRbMqTBHtsRHRmg",
  "peer_id": "12D3KooWHUDCgJ6E6J6BiWayxDjH4JxjW6nUDhrt13pncrUWYXyH"
}
"/tmp/5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH/backup-stash/0" removed.
"/tmp/5D34dL5prEUaGNQtPPZ3yN5Y6BnkfXunKXXz6fo7ZJbLwRRH/chains/a0dnet1/db/full" removed.
"/tmp/5GBNeWRhZc2jXu7D55rBimKYDk8PGk8itRYFTPfC8RJLKG5o/backup-stash/0" removed.
"/tmp/5GBNeWRhZc2jXu7D55rBimKYDk8PGk8itRYFTPfC8RJLKG5o/chains/a0dnet1/db/full" removed.
"/tmp/5Dfis6XL8J2P6JHUnUtArnFWndn62SydeP8ee8sG2ky9nfm9/chains/a0dnet1/db/full" did not exist.
"/tmp/5F4H97f7nQovyrbiq4ZetaaviNwThSVcFobcA5aGab6167dK/chains/a0dnet1/db/full" did not exist.
```

Reply:
For `N_NON_VALIDATORS = 0` (the default) we will have (for `-v 2`):
```sh
for i in $(seq 2 1); do
```
but we have generated keys (`account_ids`) for:
```sh
for i in $(seq 0 1)
```
So we have keys for indexes `0` and `1` but we would bootstrap nodes starting from `2`.